docs/extensibility/extend-rest-api-formatters.md

```diff
@@ -0,0 +1,95 @@
+# Formatters
+As part of ExtendRestAPI, we introduced `Formatters`, these are utility classes that allow you to format values to
+so that they are compatible with the StoreAPI, values such as money, currency, or HTML.
+
+## How to use them
+To get a formatter, you can use the `get_formatter` method of the `ExtendRestApi` class. This method accepts a string,
+which is the name of the formatter you want to use, e.g. (money, html, currency).
+
+This returns a `FormatterInterface` which has the `format` method.
+
+The `format` method signature is:
+```php
+format( $value, array $options = [] );
+``` 
+Only `MoneyFormatter`'s behaviour can be controlled by the `$options` parameter. 
+
+## MoneyFormatter
+The [`MoneyFormatter`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/src/StoreApi/Formatters/MoneyFormatter.php)
+class can be used to format a monetary value using the store settings. The store settings may be overriden by passing
+options to this formatter's `format` method.
+
+### Arguments
+- `$value` - `number` - The number you want to format into a monetary vaue
+- `$options` - `Array`
+  - `decimals` - `integer` - Used to control how many decimal places should be displayed in the monetary value. Defaults
+  to the store setting.
+  - `rounding_mode` - `integer` - Used to determine how to round the monetary value. This should be one of the PHP
+  rounding modes described in the
+  [PHP round() documentation](https://www.php.net/manual/en/function.round.php). Defaults to `PHP_ROUND_HALF_UP`
+  
+### Example use and returned value
+```php
+get_formatter( 'money' )->format( 10.443, [
+  'rounding_mode' => PHP_ROUND_HALF_DOWN,
+  'decimals'      => 2
+] );
+```
+returns `1044`
+
+## CurrencyFormatter
+This formatter takes an array of prices, and returns the same array but with currency data added. The currency data
+added is:
+- `currency_code` - The string representation of the currency, e.g. GPB or USD            
+- `currency_symbol` - The symbol of the currency, e.g. &pound; or &dollar;           
+- `currency_minor_unit` - How many decimal places will be shown in the currency       
+- `currency_decimal_separator` - The string used to separate the whole value and the decimal value in the currency.
+- `currency_thousand_separator` - The string used to separate thousands in the currency, for example: &pound;10,000 or &euro;10.000
+- `currency_prefix` - A string that should appear before the value.
+- `currency_suffix` - A string that should appear after the value.
```

Comment:
Do these values have defaults? If so I'd also write them here. Maybe a table would be better in this case for better structuring.

Reply:
The values added here depend entirely on your store's locale setting, so I don't think adding defaults would be very informative. I've made it into a table though :)